src/table/components/head/ListBoxWrapper.tsx

```diff
@@ -21,7 +29,13 @@ export const ListBoxWrapper = ({ children, embed, layout, columnIndex }: ListBox
   useEffect(() => {
     if (!layout || !embed) return;
 
-    embed.field(layout.qHyperCube.qDimensionInfo[columnIndex]?.qFallbackTitle).then((instance) => {
+    const fieldId = getFieldId(layout, columnIndex);
+
+    if (fieldId === undefined) {
+      return;
+    }
+
+    embed.field(fieldId).then((instance) => {
```

Comment:
`embed.field` throws an error if `fieldId` is undefined, so had to add a guard for it